book.bib

```diff
@@ -191,3 +191,140 @@ @article{zscheischler
   year={2020},
   publisher={Elsevier}
 }
+
+@article{hurrell2003,
+  title={An overview of the North Atlantic oscillation},
+  author={Hurrell, James W and Kushnir, Yochanan and Ottersen, Geir and Visbeck, Martin},
+  journal={Geophysical Monograph-American Geophysical Union},
+  volume={134},
+  pages={1--36},
+  year={2003},
+  publisher={AGU AMERICAN GEOPHYSICAL UNION}
+}
+
+@article{hurrell1997,
+  title={Decadal variations in climate associated with the North Atlantic Oscillation},
+  author={Hurrell, James W and Van Loon, Harry},
+  journal={Climatic change},
+  volume={36},
+  number={3},
+  pages={301--326},
+  year={1997},
+  publisher={Springer}
+}
+
+@article{ottersen,
```

Comment:
Add years to all of your author fields please

Reply:
done